addons/crm/models/crm_lead.py

```diff
@@ -1549,12 +1549,14 @@ def _message_post_after_hook(self, message, msg_vals):
             # we consider that posting a message with a specified recipient (not a follower, a specific one)
```

Comment:
Commit message:

"As 'email_normalized' now supported multi-emails"

-> now supports?

(For commit "[IMP] various: support multi-emails in '_message_post_after_hook'").